python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -329,26 +326,17 @@ def __deepcopy__(self, memo=None):
         # Due to problems on Hammercloud due to uncopyable object lets
         # explicitly stop these objects going anywhere near the __deepcopy__
 
-        cls = type(self)
-        c = Job.__new__(cls)
+        c = Job.__class__()
```

Comment:
It's not clear from the excerpt above but the code now has:
```python
c = Job.__class__()
c.__init__()
```
and the explicit call to `__init__` is not needed.

Regardless, This should not involve the proxy at all since `Job` in the code above means `class Job(GangaObject)` unless you;ve loaded anything from the GPI explicitly so just `c = Job()` should be fine.